assignments/this.js

```diff
@@ -13,14 +16,34 @@
 
 // code example for Window Binding
 
+
+
 // Principle 2
 
 // code example for Implicit Binding
 
+let MyInfo = function() {
+    this.name = 'Penny',
+    this.age = 'noneya',
+    this.daughter = 'Hope'
+};
+
+MyInfo.prototype.giveInfo = function() {
+    console.log(`${this.name}'s age is ${this.age} and she has a daughter named ${this.daughter}!`)
+}
+
+let info = new MyInfo();
+
+info.giveInfo()
+
 // Principle 3
 
 // code example for New Binding
 
 // Principle 4
 
-// code example for Explicit Binding
\ No newline at end of file
+// code example for Explicit Binding
```

Comment:
const musician = {
  name: 'Brett'
}

const skills = ['vocals', 'guitar', 'bass'];
    
 function introduction(skills1, skills2, skills3) {
   console.log(`Hello! My name is ${this.name} and these are my skills: ${skills1}, ${skills2}, ${skills3}`);
 }
    
 introduction.apply(musician, skills);